jobs/data/repolist.txt

```diff
@@ -27,6 +27,7 @@ dotnet/cli branch=rel/1.1.0 server=dotnet-ci
 dotnet/cli branch=release/2.0.0 server=dotnet-ci
 dotnet/cli branch=release/2.0-vs server=dotnet-ci
 dotnet/cli branch=release/15.5 server=dotnet-ci
+dotnet/cli branch=dev/release/2.0.0 server=dotnet-ci
```

Comment:
Do you want to go ahead and add the other repos to dotnet-ci as well? Or do you prefer to do so as we create the branches there?